src/components/dialog/ConfirmDialog.tsx

```diff
@@ -31,6 +31,7 @@ export function ConfirmDialog(props: ConfirmDialogProps) {
 
   return (
     <Dialog
+      shouldReturnFocusOnClose={false}
```

Comment:
Not sure about this one. I think it should be up to the application to pass this prop, and usually confirm dialogs shouldn't come from elements that have a tooltip.

Reply:
Updated thank you